packages/legend-art/src/input/Input.tsx

```diff
@@ -41,7 +41,7 @@ export const InputWithInlineValidation: React.FC<
         })}
         {...inputProps}
       />
-      {showEditableIcon && (
+      {showEditableIcon && !error && (
```

Comment:
confused about this change in general and what we are doing here. 
first we ensure !error but use error in the component. 
Second do we want this, i think we should make the contract of the component clear to the users. 
If the input has an error shouldnt the edit icon still show to allow users to be able to edit it and fix the input. Regardless seems like we need to make this component clearer